docs/glossary/index.md

```diff
@@ -0,0 +1,37 @@
+# Glossary
+
+- **Bootstrapping**: Process by which an M3DB node is brought up. Bootstrapping consists of determining
+the integrity of data that the node has, replay writes from the commit log, and/or stream missing data
+from its peers.
+
+- **Cardinality**: The number of unique metrics within the M3DB index. Cardinality increases with the
+number of unique tag/value combinations that are being emitted. 
+
+- **Datapoint**: A single timestamp/value. Timeseries are composed of multiple datapoints and a series
+of tag/value pairs. 
+
+- **Labels**: Pairs of descriptive words that give meaning to a metric. `Tags` and `Labels` are interchangeable terms.
+
+- **M3**: Highly scalable, distributed metrics platform that is comprised of a native, distributed time
+series database, a highly-dynamic and performant aggregation service, a query engine, and other
+supporting infrastructure.
+
+- **M3Coordinator**: A service within M3 that coordinates reads and writes between upstream systems,
+such as Prometheus, and downstream systems, such as M3DB.
+
+- **M3DB**: Distributed time series database influenced by [Gorilla](http://www.vldb.org/pvldb/vol8/p1816-teller.pdf)
+and [Cassandra](http://cassandra.apache.org/) released as open source by Uber Technologies.
+
+- **M3Query**: A distributed query engine for M3DB. Unlike M3Coordinator, M3Query only provides supports for reads.
```

Comment:
m3q and m3c only differ by configs right?

Reply:
I don't think we need to specify that here.

Reply:
M3Query in my use can also be used for changing configuration of placements/adding nodes, namespaces.